exporter/collector/internal/integrationtest/inmemoryocexporter.go

```diff
@@ -16,85 +16,108 @@ package integrationtest
 
 import (
 	"context"
-	"fmt"
-	"sort"
 	"time"
 
-	"go.opencensus.io/metric/metricdata"
+	"contrib.go.opencensus.io/exporter/stackdriver"
 	"go.opencensus.io/metric/metricexport"
 	"go.opencensus.io/plugin/ocgrpc"
 	"go.opencensus.io/stats/view"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
 
 	"github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector"
 )
 
-var _ metricexport.Exporter = (*InMemoryOCExporter)(nil)
-
 // OC stats/metrics exporter used to capture self observability metrics
 type InMemoryOCExporter struct {
-	c      chan []*metricdata.Metric
-	reader *metricexport.Reader
+	testServer          *MetricsTestServer
+	reader              *metricexport.Reader
+	stackdriverExporter *stackdriver.Exporter
 }
 
-func (i *InMemoryOCExporter) ExportMetrics(ctx context.Context, data []*metricdata.Metric) error {
-	i.c <- data
-	return nil
+func getViews() []*view.View {
+	views := []*view.View{}
+	views = append(views, collector.MetricViews()...)
+	// TODO: enable this
+	// views = append(views, ocgrpc.DefaultClientViews...)
```

Comment:
Should this TODO be fixed in this PR?

Reply:
I'm gonna do in the next PR along with the normalization of the values for the problematic ocgrpc metrics